src/tokens/tokens.interfaces.ts

```diff
@@ -154,8 +195,9 @@ export class TokenTransfer {
   requestId?: string;
 }
 
-export class TokenMint extends OmitType(TokenTransfer, ['from']) {}
-export class TokenBurn extends OmitType(TokenTransfer, ['to']) {}
```

Comment:
I don't think these types should be changed. We should keep the connector interface the same - if certain mint/burn/transfer operations are not authorized, it should be decided by the smart contract.